### DISCUSSION TITLE
DBeaver Export ALL records in a dataset.

### DISCUSSION THREAD

**Discussion by Shortmeister1** [dbeaver/dbeaver#22107] [Q&A]
Hi

I am trying to export more than 200 records from DBeaver Community to a CSV. I have no idea how to do this in the Export Wizard. I know I can change the onscreen dataset size, but sure there must be a way of defaulting the export to ALL RECORDS?

Thanks
Martin

**Comment by LonwoLonwo** [comment]
Hello @Shortmeister1 

Did you try to use these export tools? https://github.com/dbeaver/dbeaver/wiki/Data-transfer#export-data
